tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Services/ContentPublishingServiceTests.Publish.cs

```diff
@@ -46,7 +46,7 @@ public async Task Can_Publish_Child_Of_Root()
     [TestCase(false)]
     public async Task Publish_Branch_Does_Not_Publish_Unpublished_Children_Unless_Explicitly_Instructed_To(bool force)
```

Comment:
Updated testcase to use PublishBranchForceOptions, checking all permutations against a "unpublishedChildrenShouldBePublished" boolean.
```
[TestCase(PublishBranchForceOptions.None,false)]
[TestCase(PublishBranchForceOptions.PublishUnpublished,true)]
[TestCase(PublishBranchForceOptions.ForceRepublish,false)]
[TestCase(PublishBranchForceOptions.ForceAll,true)]
```

Reply:
Have added these test cases.